docs/docs/guides/web3_plugin_guide/plugin_authors.md

```diff
@@ -98,14 +93,9 @@ Below is representing a plugin user's code that does not configure an Ethereum p
 
 ```typescript
 // registering_a_plugin.ts
-import { Web3Context } from './web3_export_helper';
-import { CustomRpcMethodsPlugin } from './custom_rpc_methods_plugin';
+import { Web3Context } from 'web3-core';
 
-declare module 'web3-core' {
-	interface Web3Context {
-		customRpcMethods: CustomRpcMethodsPlugin;
-	}
-}
+import { CustomRpcMethodsPlugin } from './custom_rpc_methods_plugin';
 
 const web3Context = new Web3Context();
 web3Context.registerPlugin(new CustomRpcMethodsPlugin());
```

Comment:
No it's meant to be where it is. It's adding additional context for the plugin author showing how their code could be affected by what the user does/n't do (e.g. this example shows a provider error being thrown from the plugin author's code because the plugin user didn't provide a provider). [This](https://github.com/web3/web3.js/pull/5621/commits/96c90445070f2751a1286597cd40fdc24b3d421e) commit does reword this section a little